integration-tests/src/tests/client/features/wallet_contract.rs

```diff
@@ -48,24 +45,6 @@ fn check_tx_processing(
     next_height
 }
 
-fn view_request(env: &TestEnv, request: QueryRequest) -> QueryResponse {
-    let head = env.clients[0].chain.head().unwrap();
-    let head_block = env.clients[0].chain.get_block(&head.last_block_hash).unwrap();
-    env.clients[0]
-        .runtime_adapter
-        .query(
-            ShardUId::single_shard(),
-            &head_block.chunks()[0].prev_state_root(),
-            head.height,
-            0,
-            &head.prev_block_hash,
-            &head.last_block_hash,
-            head_block.header().epoch_id(),
-            &request,
-        )
-        .unwrap()
-}
```

Comment:
This change seems to be unrelated to the implementation of non-refundable storage. Would it be possible to split it out into its own NFC PR?